lib/chef-dk/command/verify.rb

```diff
@@ -96,6 +96,20 @@ def components
         c.smoke_test { run_in_tmpdir("kitchen init") }
       end
 
+      # We only ship delivery-cli on *nix only ATM
+      unless Chef::Platform.windows?
+        add_component "delivery-cli" do |c|
+          # There is no base_dir as delivery-cli is a single binary in
+          # `/opt/chefdk/bin`.
+          c.base_dir = "chef-dk"
+          c.smoke_test do
+            tmpdir do |cwd|
+              sh!("delivery setup --user=shipit --server=delivery.shipit.io --ent=chef --org=squirrels", cwd: cwd)
```

Comment:
This is probably OK for now; pretty much everything else requires a server to communicate with (though it may be possible to set up a `delivery job` run as a smoke test; I'll have to take a look at that). 

We're going to have some Cucumber tests soon that can be used for integration tests, which take care of stubbing those external dependencies, and will give much more confidence that the thing actually works like we want.

For now, though, this is totally good enough.